roles/extract_operator_bundle/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+- name: "Ensure that the operator bundle directory is empty"
+  file:
+    state: "absent"
+    path: "{{ operator_bundle_dir }}"
+  when: operator_bundle_dir is defined
+
+- name: "Ensure that the operator testing directory exists and is empty"
+  file:
+    state: "{{ item }}"
+    path: "{{ operator_work_dir }}"
+  with_items:
+    - absent
+    - directory
+  when: operator_work_dir is defined
+
+- name: "Copy the bundle image layers into a local directory using skopeo"
+  shell: "skopeo copy docker://{{ bundle_image }} dir:{{ operator_bundle_dir }} -f oci"
+  register: skopeo_copy_result
+
+- name: "Check that the image manifest.json file exists"
+  stat:
+    path: "{{ operator_bundle_dir }}/manifest.json"
+  register: image_manifest_stat_result
+
+- name: "Fail if the manifest.json file doesn't exist"
+  fail:
+    msg: "Failing because the manifest.json file doesn't exist"
+  when: not image_manifest_stat_result.stat.exists
+
+- name: "Parse the image manifest.json"
+  set_fact:
+    image_manifest: "{{ lookup('file', \"{{ operator_bundle_dir }}/manifest.json\") | from_json }}"
```

Comment:
We can directly use skope inspect. It is simpler and provide the same data
```
sh# skopeo inspect --raw oci:/tmp/data/oci | jq .
{ 
  "schemaVersion": 2,
  "config": {
    "mediaType": "application/vnd.oci.image.config.v1+json",
    "digest": "sha256:1e79bfc95fd787192af6ac84e95735f80c7506bf1a1e973bb5a5805b75bb92d1",
    "size": 1687
  },
  "layers": [
    {
      "mediaType": "application/vnd.oci.image.layer.v1.tar+gzip",
      "digest": "sha256:07aec32d6429e630c6b4d245eecb35b25a3748f78c5bccd07e061f285dcebf4f",
      "size": 7504
    }
  ]
} 

sh# skopeo inspect --raw dir:/tmp/data/dir | jq .
{  
  "schemaVersion": 2,
  "config": {
    "mediaType": "application/vnd.oci.image.config.v1+json",
    "digest": "sha256:1e79bfc95fd787192af6ac84e95735f80c7506bf1a1e973bb5a5805b75bb92d1",
    "size": 1687
  },
  "layers": [
    {
      "mediaType": "application/vnd.oci.image.layer.v1.tar+gzip",
      "digest": "sha256:07aec32d6429e630c6b4d245eecb35b25a3748f78c5bccd07e061f285dcebf4f",
      "size": 7504
    }
  ]
}  

sh# jq . /tmp/data/dir/manifest.json
{
  "schemaVersion": 2,
  "config": {
    "mediaType": "application/vnd.oci.image.config.v1+json",
    "digest": "sha256:1e79bfc95fd787192af6ac84e95735f80c7506bf1a1e973bb5a5805b75bb92d1",
    "size": 1687
  },
  "layers": [
    {
      "mediaType": "application/vnd.oci.image.layer.v1.tar+gzip",
      "digest": "sha256:07aec32d6429e630c6b4d245eecb35b25a3748f78c5bccd07e061f285dcebf4f",
      "size": 7504
    }
  ]
}
```

Reply:
Yep, changed.